src/browser/api_protocol/api_handlers/authorization.js

```diff
@@ -162,7 +162,7 @@ function addPendingAuthentication(uuid, token, file, sponsor, authReqPayload) {
 }
 
 function authenticateUuid(authObj, authRequest, cb) {
-    if (ExternalApplication.getExternalConnectionByUuid(authRequest.uuid)) {
+    if (ExternalApplication.getExternalConnectionByUuid(authRequest.uuid) || coreState.getAppByUuid(authRequest.uuid)) {
```

Comment:
Hey @weiteho, just want to make sure we take into account the re-connecting scenario, can you confirm that the  core cleans the map on a disconnect ? 

In theory we should be able to connect to a runtime then disconnect and re-connect successfully.

Reply:
Yes, it's handled already when the connection to server is closed.

Reply:
👍  awesome.

Reply:
+ a test for such scenario?